docs/resources/cdn_resource.md

```diff
@@ -415,6 +417,27 @@ Optional:
 
 - `enabled` (Boolean)
 
+<a id="nestedblock--options--proxy_connect_timeout"></a>
+### Nested Schema for `options.proxy_connect_timeout`
+
+Required:
+
+- `value` (Map of String) Specify time in seconds ('1s', '5s' for example).
```

Comment:
@MarianaStrix please also regenerate docs with the correct types, it shouldn't be a `Map of String`